src/ActivitiesSDKAdapter.js

```diff
@@ -169,29 +179,51 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
    * @returns {Observable.<Activity>} Observable that emits the posted activity (including id)
    */
   postActivity(activity) {
-    logger.debug('ACTIVITY', undefined, 'postActivity()', ['called with', {activity}]);
+    let activity$;
     const card = this.getAdaptiveCard(activity);
 
-    const object = card && {
-      cards: [JSON.stringify(card)],
-      displayName: activity.text,
-    };
+    if (card) {
```

Comment:
const activityToCreate$ = this.hasAdaptiveCard(...) ? from(...) : of(activity);

return activityToCreate$.pipe(
  concatMap(activityToCreate => convesation.post(...)),
  fromSDKActivity,

Reply:
or just activity$ instead of activityToCreate$